internal/testrunner/runners/pipeline/test_result.go

```diff
@@ -103,24 +104,37 @@ func compareJsonNumbers(a, b json.Number) bool {
 }
 
 func diffJson(want, got []byte) (string, error) {
-	opts := jsondiff.DefaultConsoleOptions()
-
-	// Remove colored output.
-	opts.Added = jsondiff.Tag{Begin: "+ "}
-	opts.Removed = jsondiff.Tag{Begin: "- "}
-	opts.Changed = jsondiff.Tag{}
-	opts.Skipped = jsondiff.Tag{}
-
-	// Configure diff.
-	opts.SkipMatches = true
-	opts.CompareNumbers = compareJsonNumbers
+	var gotVal, wantVal interface{}
+	err := jsonUnmarshalUsingNumber(want, &wantVal)
+	if err != nil {
+		return "", fmt.Errorf("invalid want value: %w", err)
+	}
+	err = jsonUnmarshalUsingNumber(got, &gotVal)
+	if err != nil {
+		return "", fmt.Errorf("invalid got value: %w", err)
+	}
+	if cmp.Equal(gotVal, wantVal, cmp.Comparer(compareJsonNumbers)) {
+		return "", nil
+	}
 
-	result, diff := jsondiff.Compare(want, got, &opts)
```

Comment:
We could keep this comparison method even if later we change the diff that we print, right?

Reply:
Yes, that's right.